frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts

```diff
@@ -1,10 +1,10 @@
 import { execSync } from 'child_process';
-import * as _ from 'lodash';
-import { browser, ExpectedConditions as until } from 'protractor';
 import { click } from '@console/shared/src/test-utils/utils';
 import { isNodeReady } from '@console/shared/src/selectors/node';
 import { PodKind } from '@console/internal/module/k8s';
 import { getName } from '@console/shared/src/selectors/common';
+import { browser, ExpectedConditions as until } from 'protractor';
+import * as _ from 'lodash';
```

Comment:
We have been using the eslint import plugin for a while now, but for some reason it hasn't really enforced sort order. This change happened when using the `--fix` option. Not sure why the sort order rule was not enforced prior to this update.

Reply:
Do you know the particular rule that caused this?

Reply:
I feel like it would be this rule: `sort-imports`
But I believe it's off in the air-bnb config.
I think if we can address whatever rule caused this change, it would be good imo.
I'd like to address changing rules on a per needs basis and a single PR to get feedback.

Reply:
Yeah, sounds good to me. I can disable that rule in our overrides to prevent the change

Reply:
Fixed